openwisp_notifications/templates/admin/base_site.html

```diff
@@ -15,18 +15,17 @@
   {% endif %}
 {% endblock %}
 
-{% block userlinks %}
-  <div class="ow-notifications" id="openwisp_notifications">
-    <button id="ow-notification-btn">
-      <img class="bell-icon" src="{% static 'openwisp-notifications/images/icons/icon-bell.svg' %}" alt="{% trans 'bell icon' %}" />
-    </button>
+{% block user-tools %}
+  {{ block.super }}
+  <div class="ow-notifications toggle-btn" id="openwisp_notifications">
+    <button id="ow-notification-btn"></button>
     {% unread_notifications %}
   </div>
   {% csrf_token %}
   {% notification_toast %}
   {% notification_widget %}
-{% endblock %}
-
+{% endblock  %}
```

Comment:
Change not required.
P.S If you haven't already, install linters and if you use vscode, add:
```
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true,
```

to ensure file ends with a new line and trailing whitespaces are removed!

Reply:
Thank you :smile: